doc/sphinx/proofs/automatic-tactics/auto.rst

```diff
@@ -179,31 +184,12 @@ Programmable proof search
 
    This tactic may loop if you build non terminating rewriting systems.
 
-.. tacv:: autorewrite with {+ @ident} using @tactic
-
-  Performs, in the same way, all the rewritings of the bases :n:`{+ @ident}`
-  applying tactic to the main subgoal after each rewriting step.
-
-.. tacv:: autorewrite with {+ @ident} in @qualid
-
-   Performs all the rewritings in hypothesis :n:`@qualid`.
-
-.. tacv:: autorewrite with {+ @ident} in @qualid using @tactic
-
-  Performs all the rewritings in hypothesis :n:`@qualid` applying :n:`@tactic`
-  to the main subgoal after each rewriting step.
-
-.. tacv:: autorewrite with {+ @ident} in @goal_occurrences
-
-  Performs all the rewriting in the clause :n:`@goal_occurrences`.
-
 .. seealso::
 
    :ref:`Hint-Rewrite <hintrewrite>` for feeding the database of lemmas used by
    :tacn:`autorewrite` and :tacn:`autorewrite` for examples showing the use of this tactic.
```

Comment:
We should also add a link to the "Strategies for rewriting" section of "Generalized rewriting". `rewrite_strat` is supposed to be a better `autorewrite`. Eventually, we might want to move the documentation of the two closer together.